orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/TitusInterestingHealthProviderNamesSupplier.java

```diff
@@ -34,8 +36,8 @@
 public class TitusInterestingHealthProviderNamesSupplier implements InterestingHealthProviderNamesSupplier {
   private static final String TITUS = "titus";
   private static final String INTERESTING_HEALTH_PROVIDER_NAMES = "interestingHealthProviderNames";
-  private static final String ROLLING_PUSH = "rollingpush";
-  private static final String SUPPORTED_STAGE = "rollingPush";
+  private static final List<String> SUPPORTED_STRATEGIES = Collections.singletonList("rollingpush");
```

Comment:
I am inclined to agree with you on that it can be applied to all strategies.
With the side effect being that `interestingHealthProviderNames` resolved here are the source of truth.
I will run some test locally to confirm.